### DISCUSSION TITLE
Where is the EFI folder on the pen drive?

### DISCUSSION THREAD

**Discussion by bjwil1991** [valnoxy/t480-oc#33] [Q&A]
I followed the steps until I got to the spot that says copy the EFI folder to the EFI partition and my 32GB pendrive does not have that listed and I used my Windows laptop to format the drive.

Was there a step I missed or would I use a Mac to create an EFI partition?

**Comment by valnoxy** [comment]
Hey,
you need to rename the folder (e.g.) 'EFI - Monterey' to EFI and then copy that folder to the pen drive.
